### PR TITLE
WIP: gcp flake testing

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

File: vendor/k8s.io/kubernetes/cmd/hyperkube/main.go [hunk 1]
```diff
@@ -57,6 +57,7 @@ func main() {
 	pflag.CommandLine.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 	// cliflag.InitFlags()
+	pflag.Set("log-file", "/etc/kubernetes/static-pod-resources/kcm.log")
```
Comment (smarterclayton) [inline vendor/k8s.io/kubernetes/cmd/hyperkube/main.go]: does this get stomped by the new version?  Or will it carry over (can multiple glogs write to the same file?)

Comment (ironcladlou) [reply vendor/k8s.io/kubernetes/cmd/hyperkube/main.go]: I'm just trying to get logs written somewhere at all right now, and once I know that works, I'm going to make my 4.3 PR read and dump the file the 4.2 code created

File: vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/gce/gce_loadbalancer.go [hunk 2]
```diff
@@ -107,14 +108,23 @@ func (g *Cloud) GetLoadBalancerName(ctx context.Context, clusterName string, svc
 
 // EnsureLoadBalancer is an implementation of LoadBalancer.EnsureLoadBalancer.
 func (g *Cloud) EnsureLoadBalancer(ctx context.Context, clusterName string, svc *v1.Service, nodes []*v1.Node) (*v1.LoadBalancerStatus, error) {
```
Comment (abhat) [inline vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/gce/gce_loadbalancer.go]: So this should really go in the 4.3 PR then?

Comment (ironcladlou) [reply vendor/k8s.io/kubernetes/pkg/cloudprovider/providers/gce/gce_loadbalancer.go]: Yeah, first I'm just trying to see if my log redirection is even working at all

**Pull request closed without merging**